SampleTransfers/StructMapTransferSamples/SimpleBookExample/metadata/mets_structmap.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="utf-8"?>
+<mets:mets xmlns:mets="http://www.loc.gov/METS/">
+  <mets:structMap TYPE="logical">
+    <mets:div TYPE="book" LABEL="How to create a hierarchical book">
+      <mets:div TYPE="page" LABEL="Cover">
+        <mets:fptr FILEID="cover.jpg" CONTENTIDS="objects/cover.jpg"/>
+      </mets:div>
+      <mets:div TYPE="page" LABEL="Inside cover">
+        <mets:fptr FILEID="inside_cover.jpg" CONTENTIDS="objects/inside_cover.jpg"/>
+      </mets:div>
+      <mets:div TYPE="chapter" LABEL="Chapter 1">
+        <mets:div TYPE="page" LABEL="Page 1">
+          <mets:fptr FILEID="page_01.jpg" CONTENTIDS="objects/page_01.jpg"/>
+        </mets:div>
+        <mets:div TYPE="subchapter" LABEL="Subchapter 1.1">
+          <mets:div TYPE="page" LABEL="Page 2">
+            <mets:fptr FILEID="page_02.jpg" CONTENTIDS="objects/page_02.jpg"/>
+          </mets:div>
+          <mets:div TYPE="page" LABEL="Page 3">
+            <mets:fptr FILEID="page_03.jpg" CONTENTIDS="objects/page_03.jpg"/>
+          </mets:div>
+          <mets:div TYPE="page" LABEL="Page 4">
+            <mets:fptr FILEID="page_04.jpg" CONTENTIDS="objects/page_04.jpg"/>
+          </mets:div>
+          <mets:div TYPE="subchapter" LABEL="Subchapter 1.2">
+            <mets:div TYPE="page" LABEL="Page 5">
+              <mets:fptr FILEID="page_05.jpg" CONTENTIDS="objects/page_05.jpg"/>
+            </mets:div>
+            <mets:div TYPE="page" LABEL="Page 6">
+              <mets:fptr FILEID="page_06.jpg" CONTENTIDS="objects/page_06.jpg"/>
+            </mets:div>
+            <mets:div TYPE="page" LABEL="Page 7">
+              <mets:fptr FILEID="page_07.jpg" CONTENTIDS="objects/page_07.jpg"/>
+            </mets:div>
+          </mets:div>
+          <!-- Subchapter 1.2 -->
```

Comment:
I'm curious as to why Subchapter 1.2 is nested inside Subchapter 1.1 instead of appearing at the same hierarchical level (i.e. as a sibling level)?

Reply:
Great spot. That was an oversight on my part, quite literally, I couldn't differentiate the levels. I have added a new commit to correct each of the three book examples. I'll submit for re-review to get the thumbs up before merging.